wallet-electron-ts/makeBin.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Preconditions
+# Need to run `npm run build` on monorepo root
+# Need to run ./buildDependencies.sh
+# bitcoin-s-server.zip is in wallet-electron/
+
+# Delete old /bin
+rm -rf bin
+
+# Make /bin
+mkdir -p bin
+
+# Copy common-ts
+#mkdir -p bin/common-ts
+#cp -a ../common-ts/dist/cjs/. bin/common-ts
+
+# Copy wallet-ts
+#mkdir -p bin/wallet-ts
+#cp -a ../wallet-ts/dist/cjs/. bin/wallet-ts
+
+# Copy wallet-server-ui
+mkdir -p bin/wallet-server-ui
+cp -a ../wallet-server-ui/dist/wallet-server-ui/. bin/wallet-server-ui
+
+# Copy wallet-server-proxy
+mkdir -p bin/wallet-server-ui-proxy
+cp -a ../wallet-server-ui-proxy/dist/cjs/build.json . # for `npm run start`
+cp -a ../wallet-server-ui-proxy/dist/cjs/build.json bin/wallet-server-ui-proxy
+cp -a ../wallet-server-ui-proxy/dist/bundle-static.js bin/wallet-server-ui-proxy
+# Replace config.json
+cp -a proxy-config.json bin/wallet-server-ui-proxy/config.json
+
+# Expand bitcoin-s-server zip
+echo "TODO : Pull down bitcoin-s-server.zip"
+mkdir -p bin/bitcoin-s-server
+unzip bitcoin-s-server.zip -d bin/bitcoin-s-server
+chmod +x bin/bitcoin-s-server/bin/bitcoin-s-server
```

Comment:
I think this is an accidental duplicate `bin/bitcoin-s-server/bin/bitcoin-s-server`

![Screenshot from 2022-05-09 13-11-57](https://user-images.githubusercontent.com/3514957/167471297-3556a6a9-3e2b-4cf6-b843-3ead2995fbd2.png)

Reply:
How do you mean? Where does it live? If you mean the one in .webpack/, that is the final bin folder that is built for the Electron app.

Reply:
I guess it seems to me that `bin/bitcoin-s-server/bin/bitcoin-s-server` is redundant, but I think the root problem was fixed in a19aac11e334a12310f11ed3903f134e13d62e80 so this is just a naming issue. I thought there was something deeper, i'm resolving this thread for now